Alpha prototype entry added (#227)
DFE-Digital/bat-design-history: app/posts/register-trainee-teachers/2020-09-07-register-teacher-trainees-alpha-prototype.md

---

> title: Initial alpha static prototype 

Reviewer: If the project is in beta now, this could be "Alpha prototype"

Author: Another good shout. Will update next pass.

> title: Register Trainee Teachers - alpha prototype 

Reviewer 2: ```suggestion
title: Initial alpha static prototype 
```

Register Train Teachers isn’t necessary as already in that section. Maybe distinguish this prototype from the functional one you are developing now.

description: A look at the alpha prototype and beginning to think about the next steps.
date: 2020-09-07
related:
  items:
  - text: Alpha assessment

>     href: https://docs.google.com/presentation/d/1Ws9gmeHZTjmc2NRZlSCcVEfReEmyi9KB1_BLyyJv-ZE/edit?usp=sharing

Reviewer 2: ```suggestion
    href: https://docs.google.com/presentation/d/1Ws9gmeHZTjmc2NRZlSCcVEfReEmyi9KB1_BLyyJv-ZE/
```

  - text: Alpha research overview

>     href: https://dfedigital.atlassian.net/wiki/spaces/TTDT/pages/1626931201/3.+User+Research+Rounds+Sprint+Work

Reviewer 2: ```suggestion
    href: https://dfedigital.atlassian.net/wiki/spaces/TTDT/pages/1626931201/
```

Which part of the URL is necessary (suspect the later part may change?)

Author: Either works. The cleaner version redirects to the lonnger version.

  - text: Alpha research analysis 
    href: https://dfedigital.atlassian.net/wiki/spaces/TTDT/pages/1626931201/3.+User+Research+Rounds+Sprint+Work
  - text: Static prototype 
    href: https://projects.invisionapp.com/share/R2YGO89WADV#/screens/428900167
---

> The Register Teacher Trainee service exists to support the exchange of data on trainee teachers in England between accredited Initial Teacher Training (ITT) providers and the DfE.

Reviewer 2: ```suggestion
The Register teacher trainee service exists to support the exchange of data on trainee teachers in England between accredited Initial Teacher Training (ITT) providers and the DfE.
```

Service names use sentence case.

> Each year the service provides data on approximately 40K trainee teachers across 632 institutions in England.

Reviewer 2: ```suggestion
Each year the service provides data on approximately 40,000 trainee teachers across 632 institutions in England.
```

Avoid abbreviations.

The Register alpha focused on replacing an existing DfE product, the [Database of trainee teachers and providers (DTTP)](https://www.gov.uk/guidance/database-of-trainee-teachers-and-providers-dttp) and ran from 27 April to the end of July, with an assessment in early August.

## Next steps

The alpha produced a [static prototype hosted on InVision](https://projects.invisionapp.com/share/R2YGO89WADV#/screens/428900167). 

A functional prototype is being built which will enable us to gather further insights from research.

> The team has identified areas in the current prototype that will require further thought for the functional prototype such as:

Reviewer: ```suggestion
The team has identified areas in the static prototype that will require further thought such as:
```
